packages/editor/src/components/properties/PropertiesPanelContainer.tsx

```diff
@@ -79,14 +80,15 @@ export const PropertiesPanelContainer = () => {
       Object.values(materialLibrary.materials.value).find(({ material }) => material.uuid === nodeEntity)
     : isObject3D
     ? world.scene.getObjectByProperty('uuid', nodeEntity as string)
-    : world.entityTree.entityNodeMap.get(nodeEntity as Entity)
+    : nodeEntity
 
   if (!nodeEntity || !node) {
     content = <NoNodeSelectedMessage>{t('editor:properties.noNodeSelected')}</NoNodeSelectedMessage>
   } else if (isObject3D) {
     content = (
       <StyledNodeEditor>
-        <Object3DNodeEditor multiEdit={multiEdit} node={node as EntityTreeNode} />
+        {/* @todo these types are incorrect */}
+        <Object3DNodeEditor multiEdit={multiEdit} entity={node as Entity} />
```

Comment:
Object3DNodeEditor needs more typing, but I believe the entity property it expects is an Object3D, not an Entity.